code/datums/datacore.dm

```diff
@@ -160,6 +160,8 @@
 		"Service" = GLOB.service_positions,
 		"Silicon" = GLOB.nonhuman_positions
 	)
+	var/list/heads = GLOB.command_positions + list("Quartermaster")
```

Comment:
isnt QM part of command_positions on this codebase?

Reply:
yup, I'll fix it later maybe